graphql/database/experiments/features.js

```diff
@@ -198,6 +199,47 @@ const features = {
       },
     ],
   },
+  [SFAC_EXTENSION_PROMPT]: {
+    defaultValue: 'Control',
+    rules: [
+      /* Begin internal overrides */
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          [`internalExperimentOverrides.${SFAC_EXTENSION_PROMPT}`]: {
+            $eq: 'Control',
+            $exists: true,
+          },
+        },
+        force: 'Control',
+      },
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          [`internalExperimentOverrides.${YAHOO_SEARCH_NEW_USERS_V2}`]: {
+            $eq: 'Notification',
+            $exists: true,
+          },
+        },
+        force: 'Notification',
+      },
+      /* End internal overrides */
+      {
+        variations: ['Control', 'Notification'],
+        weights: [0.5, 0.5],
+        coverage: 0.0,
+        condition: {
+          tabs: {
+            $gt: 3,
+          },
```

Comment:
Add "v4BetaEnabled" for good measure. That's just future-proofing, in case we start fetching these features on Legacy.